examples/http_loader.rb

```diff
@@ -43,27 +43,43 @@ def initialize(host:, size: 4, timeout: 4)
       @host = host
       @size = size
       @timeout = timeout
+      @futures = {}
     end
 
-    def perform(operations)
+    def perform_on_wait(operations)
```

Comment:
Should this perhaps be:

```suggestion
    def perform_on_any_wait(operations)
```

To match the naming of `Loader#on_any_wait`